### ISSUE TITLE
[ALBERT]Has anyone reproduced ALBERT a scores on GLUE dataset?

### ISSUE THREAD

**Issue by lonePatient** [google-research/ALBERT#99]
I convert tf weight to pytorch weight ,and on QQP dataset, I only get 87% accuracy.

model: albert-base
epochs: 3
learning_rate; 2e-5
batch size: 24
max sequence length: 128
warmup_proportion: 0.1
